File: examples/python/basic_usage.py
#!/usr/bin/env python3
/**
 * Basic usage example for HiTechCloud Tools Python SDK
 * 
 * Set your API key:
 *   export HITECHCLOUD_TOOLS_API_KEY=your-api-key
 */
from hitechcloud_tools import HiTechCloudTools

client = HiTechCloudTools(api_key="your-api-key")

# GeoIP lookup
result = client.mang_and_ha_tang.geoip_lookup(ip="8.8.8.8")
print(result)

# Domain WHOIS
whois = client.ten_mien_and_ssl.domain_whois(domain="example.com")
print(whois)

# Exchange rates
rates = client.tai_chinh_and_ty_gia.exchange_rates(base="USD")
print(rates)

# Vietnamese provinces
provinces = client.du_lieu_viet_nam.provinces()
print(provinces)

File: examples/ruby/basic_usage.rb
#!/usr/bin/env ruby
/**
 * Basic usage example for HiTechCloud Tools Ruby SDK
 * 
 * Set your API key:
 *   export HITECHCLOUD_TOOLS_API_KEY=your-api-key
 */
require 'hitechcloud_tools'

client = HiTechCloudTools::Client.new(api_key: 'your-api-key')

# GeoIP lookup
result = client.mang_and_ha_tang.geoip_lookup(ip: "8.8.8.8")
puts result

# Domain WHOIS
whois = client.ten_mien_and_ssl.domain_whois(domain: "example.com")
puts whois

# Exchange rates
rates = client.tai_chinh_and_ty_gia.exchange_rates(base: "USD")
puts rates
